systemd/ocp-custom-domain.sh

```diff
@@ -2,8 +2,8 @@
 
 set -x
 
-if [ -z $CRC_CLOUD ]; then
-    echo "Not running in crc-cloud mode"
+if [ -z $CRC_SELF_SUFFICIENT ]; then
```

Comment:
i think this came back after rebasing again, we can remove this check its already part of the service `ExecCondition`?